src/applications/appeals/10182/tests/migrations/01-rename-contested-issues.unit.spec.js

```diff
@@ -1,7 +1,28 @@
 import { expect } from 'chai';
-import renameContestedIssues from '../../migrations/01-rename-contested-issues.js';
+import renameContestedIssues from '../../migrations/01-rename-contested-issues';
 
 describe('NOD rename contestedIsuses migration', () => {
+  it('should return contestedIssues empty array', () => {
```

Comment:
was this renamed `contested` from `contestable`?

Reply:
Yes, it was a migration put in place so we could move the area of disagreement code into the shared folder. Eventually, we'll move the ContestableIssuesWidget from this app into the shared folder.